app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+import datetime
+import time
+
+
+def logs():
+    while True:
+        currenttime = datetime.datetime.now()
+        name = f"app-{currenttime.strftime('%H_%M_%S')}.log"
+        with open(name, "a") as f:
+            f.writelines(str(currenttime))
```

Comment:
```suggestion
            f.write(str(currenttime))
```
Use `write` with `str` type
Use `writelines` with `list` of strings